icefoganalytics/travel-authorization
# Cleanup: Clean Up Top Level Component Pre-Profile Update

# Context

Perform some cleanup in preparation of https://github.com/icefoganalytics/travel-authorization/issues/88

# Implementation

Move some top level components into a "pages" directory, and suffix the pages with "Page".

# Testing Instructions

1. Boot the app via `dev up`.
2. Log in to the app at http://localhost:8080, and check that you get redirected to http://localhost:8080/sign-in.
3. Check that you get redirected to http://localhost:8080/dashboard after sign in.
4. Check that you can log out now, it was broken before, and that you get redirected to the /sign-in page after logout.
5. After logging back in, go to the "My Profile" page via the top right "kebab" menu.
6. Check that it works.
7. Check that you can go to the "health check" page via the kebab menu, then clicking on the version info.
